### ISSUE TITLE
Mira build failures in darshan_core_printf() function due to va_end and va_start

### ISSUE THREAD

**Issue by shanedsnyder** [darshan-hpc/darshan#254]
In GitLab by @carns on Dec 12, 2018, 18:52

From nightly tests:

```
/gpfs/mira-home/radixio/cinow-01/workspace/Radix-IO/Darshan/x86-master-profconf/install/lib/libdarshan.a(darshan-core.o): In function `darshan_core_fprintf':
/gpfs/mira-home/radixio/cinow-01/workspace/Radix-IO/Darshan/x86-master-profconf/build/darshan-runtime/../../darshan-runtime/lib/darshan-core.c:2148: undefined reference to `va_start'
/gpfs/mira-home/radixio/cinow-01/workspace/Radix-IO/Darshan/x86-master-profconf/build/darshan-runtime/../../darshan-runtime/lib/darshan-core.c:2150: undefined reference to `va_end'
collect2: error: ld returned 1 exit status
```
This doesn't happen on Ubuntu; need to determine what the difference is here.

**Comment by shanedsnyder** [comment]
In GitLab by @shanedsnyder on Dec 17, 2018, 09:56

This was actually on x86 systems, but was just related to a forgotten include related to the fprintf stuff (`stdarg.h`). Fixed and pushed to master in 66216227.

**Comment by shanedsnyder** [comment]
In GitLab by @shanedsnyder on Dec 17, 2018, 09:56

closed
